shared/actions/chat/attachment.js

```diff
@@ -59,6 +77,10 @@ function * onLoadAttachment ({payload: {conversationIDKey, messageID, loadPrevie
   // If we are loading the actual attachment,
   // let's see if we've already downloaded it as an hdPreview
   if (!loadPreview && !isHdPreview) {
+    if (existingMessageState === 'downloading') {
```

Comment:
:sheep: This bailing in the middle makes me feel uneasy. I think it's best if we can do that before even calling this.